__tests__/api.test.js

```diff
@@ -65,3 +64,40 @@ describe("GET: /api", () => {
       });
   });
 });
+describe("GET: /api/articles/1", () => {
+  test("200: should return an article by its id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.article;
+
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
```

Comment:
As we are only receiving one object, we can explicitly test its values instead of using `expect.any()`